README.md

```diff
@@ -0,0 +1,36 @@
+[![Build and Test](https://github.com/bigladder/lattice/actions/workflows/build-and-test.yaml/badge.svg)](https://github.com/bigladder/lattice/actions/workflows/build-and-test.yaml)
+
+[![Web Documentation](https://github.com/bigladder/lattice/actions/workflows/build-web.yaml/badge.svg)](https://github.com/bigladder/lattice/actions/workflows/build-web.yaml)
+
+Lattice
+===========
+
+A toolkit that provides support for a schema-based building data model framework, currently under development as ASHRAE Standard 232P. Lattice performs data model validation, intermediate schema generation, and data file validation for YAML data models compliant with ASHRAE 232P and ASHRAE 205.
```

Comment:
A python package that provides support for a schema-based data model framework, currently under development as ASHRAE Standard 232P. Lattice performs data model validation, intermediate schema generation, and data file validation for YAML data models compliant with ASHRAE 232P and ASHRAE 205.

> Lattice: an arrangement of ~atoms, ions, or molecules~ **data elements** in a ~metal or other crystalline solid~ **standardized data model structure**.

Reply:
Open to other ideas too.

Reply:
I modified 'toolkit' to 'Python package,' but not sure what you are thinking with regard to the definition of a Lattice?

Reply:
I was toying with a way of explaining where the name comes from...but I'm not sure it works well.

Reply:
A cartoon would be awesome, but I think it's a bit stilted in words.